docs/src/main/asciidoc/security-authentication-mechanisms.adoc

```diff
@@ -402,7 +402,12 @@ For more information about managing the individual tenant configurations in mult
 
 ==== OpenID Connect client and filters
 
+ifndef::no-quarkus-elytron-security-oauth2[]
 The `quarkus-oidc-client` extension provides `OidcClient` for acquiring and refreshing access tokens from OpenID Connect and OAuth2 providers that support the following token grants:
+endif::no-quarkus-elytron-security-oauth2[]
+ifdef::no-quarkus-elytron-security-oauth2[]
+The `quarkus-oidc-client` extension provides `OidcClient` for acquiring and refreshing access tokens from OpenID Connect providers that support the following token grants:
```

Comment:
This update shouldn't be done,  since many OIDC providers are also OAuth2 providers and quarkus oidc client can talk to all of them. Elytron OAuth2 extension is about providing oauth2 token verification via remote token introspection only

Reply:
Updated. Thanks for the explanation.